SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -57,18 +63,50 @@ void SpatialRPCService::ProcessChanges(const float NetDriverTime)
 		CrossServerRPCs->ProcessChanges();
 	}
 
-	if (NetDriverTime - LastProcessingTime > GetDefault<USpatialGDKSettings>()->QueuedIncomingRPCRetryTime)
+	const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
+
+	if (NetDriverTime - LastIncomingProcessingTime > Settings->QueuedIncomingRPCRetryTime)
 	{
-		LastProcessingTime = NetDriverTime;
+		LastIncomingProcessingTime = NetDriverTime;
 		ProcessIncomingRPCs();
 	}
+
+	if (NetDriverTime - LastOutgoingProcessingTime > Settings->QueuedOutgoingRPCRetryTime)
+	{
+		LastOutgoingProcessingTime = NetDriverTime;
+		ProcessOutgoingRPCs();
+	}
+}
+
+void SpatialRPCService::Flush()
```

Comment:
Perhaps this should be renamed from flush now as it doesn't always result in a flush from the worker connection, and we don't use dispatch when processing incoming updates. SendUpdates/PushUpdates/PushComponentUpdates?